svc/LocalStorageService.js

```diff
@@ -16,6 +16,18 @@ import store from 'store2';
  */
 export class LocalStorageService extends HoistService {
 
+    constructor() {
+        super();
+        errorIf(this.isFake,
+            'Local Storage is not supported in this browser. Transient in-memory storage will ' +
+            'be used for various user state, and lost when the page is closed.'
```

Comment:
the "for various user state"  part is a little confusing.
Could the whole message be:
'Local Storage is not supported in this browser. Transient in-memory storage will be used as a fallback, so data stored with this service will be lost when the page is closed.'